docs/extend/extend-apiml/api-mediation-caching-service.md

```diff
@@ -36,25 +37,30 @@ Information from cached APIs is stored as a JSON in the following format:
 ## Storage methods
 
 The Caching Service supports the following storage solutions, which provide the option to add custom implementation.  
+
+### Infinispan (recommended)
+
+Infinispan is a storage solution that can also run on the z/OS platform. It can store data structures in key-value pairs, has high-availability support, and is highly performant.
+
+For more information about the Infinispan storage access method, see [Using Infinispan as a storage solution through the Caching service](./api-mediation-infinispan.md).
+
 ### VSAM
 
 VSAM can be used to organize records into four types of data sets: key-sequenced, entry-sequenced, linear, or relative record. Use VSAM as the storage solution for production. VSAM is used primarily for applications and is not used for source programs, JCL, or executable modules. ISPF cannot be used to display or edit VSAM files.
 
 For more information about the VSAM storage access method, see [Using VSAM as a storage solution through the Caching Service](./api-mediation-vsam.md).
+
 ### Redis
 
 Redis is a common storage solution that runs outside of the z/OS platform. It can store data structures in key-value pairs, has high-availability support, and is highly performant.
```

Comment:
"A common storage solution that runs outside of the z/OS platform. Redis can store ..."

Reply:
I made a mistake when I created this branch and, as a result, it now includes another writer's content. Since this is the work of another writer, I will not be editing this file.